pallets/maintenance-mode/src/tests.rs

```diff
@@ -0,0 +1,71 @@
+use super::*;
+use crate::{mock::*, Error, ACTIVATED, DEACTIVATEDD};
+use frame_support::{assert_noop, assert_ok, traits::Contains};
+use pallet_balances::{self, Call as BalancesCall};
+use pallet_remark::{self, Call as RemarkCall};
+
+#[test]
+fn activate_maintenance_mode_works() {
+	new_test_ext().execute_with(|| {
+		assert_eq!(MaintenanceModeStatus::<Test>::get(), DEACTIVATEDD);
+		assert_ok!(MaintenanceMode::activate_maintenance_mode(RuntimeOrigin::root()));
+
+		let maintenance_mode = MaintenanceModeStatus::<Test>::get();
+		assert_eq!(maintenance_mode, ACTIVATED);
+
+		assert_noop!(
+			MaintenanceMode::activate_maintenance_mode(RuntimeOrigin::root(),),
+			Error::<Test>::MaintenanceModeAlreadyActivated
+		);
+	});
+}
+
+#[test]
+fn deactivate_maintenance_mode_works() {
+	new_test_ext().execute_with(|| {
+		// We activate maintenance mode first so we can deactivate it.
+		assert_ok!(MaintenanceMode::activate_maintenance_mode(RuntimeOrigin::root()));
+
+		assert_ok!(MaintenanceMode::deactivate_maintenance_mode(RuntimeOrigin::root()));
+
+		let maintenance_mode = MaintenanceModeStatus::<Test>::get();
+		assert_eq!(maintenance_mode, DEACTIVATEDD);
+
+		assert_noop!(
+			MaintenanceMode::deactivate_maintenance_mode(RuntimeOrigin::root(),),
+			Error::<Test>::MaintenanceModeAlreadyDeactivated
+		);
+	});
+}
+
+#[test]
+fn call_not_filtered_in_maintenance_mode() {
```

Comment:
this should be `call_filtered_in_maintenance_mode`.

you have whitelisted only `Balance`
```
pub struct RuntimeFilteredCalls;
impl Contains<RuntimeCall> for RuntimeFilteredCalls {
	fn contains(call: &RuntimeCall) -> bool {
		match call {
			RuntimeCall::Balance(_) => true,
			_ => false,
		}
	}
}
```